climate_index_collection/reductions.py

```diff
@@ -334,3 +334,22 @@ def area_mean_weighted(
     )
 
     return averaged
+
+
+def eof_weights(dobj):
+    """To have the Empirical Orthogonal Functions (EOFs) truly orthogonal, we need to take the area of the grid cells
```

Comment:
I think this "truly orthogonal" argument is not accurate: The un-weighted EOFs are also forming a complete orthogonal basis. But the covariance matrix whose eigenvectors they are is not correct, if we don't weight the grid boxes according to the sample size they are averaged over.